sunpy/time/time.py

```diff
@@ -273,24 +273,11 @@ def is_time(time_string, time_format=None):
     out : bool
         True if can be parsed by parse_time
 
-    Notes
-    -----
-        If time_string is an instance of float, then it is assumed to be in
-        unix time format.
-
     Examples
     --------
     >>> import sunpy.time
-    >>> sunpy.time.parse_time('2012/08/01')
-    datetime.datetime(2012, 8, 1, 0, 0)
-    >>> sunpy.time.parse_time('2005-08-04T00:01:02.000Z')
-    datetime.datetime(2005, 8, 4, 0, 1, 2)
```

Comment:
Do we want these examples removed?

Reply:
Yup. And proper ones were added in their place.